content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -406,7 +406,7 @@ the bundle with their own arbitrary but stable ordering.
 ClusterTrustBundle objects should be considered world-readable within the
 cluster. If your cluster uses [RBAC](/docs/reference/access-authn-authz/rbac/)
 authorization, all ServiceAccounts have a default grant that allows them to
-**get**, **list**, and **watch** all ClusterTrustBundle objects.
+**get**, **list**, and **{{< glossary_tooltip text="watch" term_id="watch" >}}** all ClusterTrustBundle objects.
```

Comment:
I wouldn't use the tooltip here; it will make the verb look different from others and that may confuse readers more than it helps.

Reply:
Ah.. ok . Noted